### DISCUSSION TITLE
Using with net7.0 and WebAssembly Wasi.Sdk target

### DISCUSSION THREAD

**Discussion by DamianReeves** [IronLanguages/ironpython3#1657] [Q&A]
Is there any roadmap for net7.0 support? Also any reason once net7.0 is available why we couldn't use the `Wasi.Sdk`?

**Comment by slozier** [comment]
.NET 7 is supported, though no plans to release a package that targets net7.0 (as they would be "identical" to the net6.0 DLLs). You should be able to use IronPython from a net7.0 project - if it doesn't work please file a bug report with details.

As for the `Wasi.Sdk` I have no idea. If it does AOT compilation I'm guessing it won't work, but I have not tried either way.
